sklearn/ensemble/forest.py

```diff
@@ -211,11 +232,17 @@ def fit(self, X, y, sample_weight=None):
 
         self.n_outputs_ = y.shape[1]
 
-        y = self._validate_y(y)
+        y, cw = self._validate_y_cw(y)
```

Comment:
Could we have a more explicite name for what "cw" means (for the variable name and the function call).